docs/decision_record.md

```diff
@@ -6,6 +6,97 @@
 
 [TOC]
 
+## 2022-09-05 - How sequence collection are serialized prior to digestion
+
+### Decision
+
+The serialisation of a sequence collection will use the following steps
```

Comment:
I think it would be useful to add a quick definition of 'serialisation' here since I think this word can have multiple meanings in different contexts, and here I think we mean a specific definition that may not be the most common or known definition. Maybe worth a bit of background on what we're trying to do here.

Reply:
Added a line for the definition